testindex/Engine.py

```diff
@@ -73,6 +74,14 @@ def run(self):
 
                 if status2:
                     flags_list.append(True)
+                    status3, dep_graph = Validators.DependencyValidator(index_path).run()
+
+                    if status3:
+                        flags_list.append(True)
+
+                    else:
+                        print "ERROR - The container dependencies are cyclic, please resolve the same."
```

Comment:
Can we add some info about cyclic nodes here, if that info is handy to retrieve, else we can print the message and leave it for author of image to figure out.